docs/json/radarr/cf/dv-hdr10.json

```diff
@@ -11,7 +11,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(HDR(10)?)\\b)(?=.*\\b(DV|DoVi)\\b)"
+        "value": "^(?=.*\\b(HDR(10)?(?!\\+))\\b)(?=.*\\b(DV|DoVi)\\b)"
```

Comment:
```suggestion
        "value": "^(?=.*\\b(HDR(10)?(?!\\+|P(?:lus)?))\\b)(?=.*\\b(DV|DoVi)\\b)"
```
cover Plus too?

Reply:
It doesn't need to be explicitly included as the word boundary prevents it from being included